### PR TITLE
Fix test

### PR DESCRIPTION
**Pull request by danghai**
PR for fixing test #247

### CODE REVIEW

File: tests/test_array.py [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-from array.longest_non_repeat import longest_non_repeat, longest_non_repeat_two
-from array.merge_intervals import Interval, merge_v2
-from array.missing_ranges import missing_ranges
-from array.move_zeros_to_end import move_zeros
-from array.plus_one import plus_one, plus_one_v2, plus_one_v3
-from array.rotate_array import rotate_v1, rotate_v2, rotate_v3
-from array.summary_ranges import summary_ranges
-from array.three_sum import three_sum
-from array.two_sum import two_sum
+from Array.delete_nth import delete_nth, delete_nth_naive
```
Comment (goswami-rahul) [inline tests/test_array.py]: a better module name would be `arrays` for it.

File: tests/test_search.py [hunk 2]
```diff
@@ -1,11 +1,11 @@
-from search.binary_search import binary_search, binary_search_recur
-from search.first_occurance import first_occurance
-from search.last_occurance import last_occurance
-from search.search_insert import search_insert
-from search.two_sum import two_sum, two_sum1, two_sum2
-from search.search_range import search_range
-from search.find_min_rotate import find_min_rotate, find_min_rotate_recur
-from search.search_rotate import search_rotate, search_rotate_recur
+from Search.binary_search import binary_search, binary_search_recur
```
Comment (goswami-rahul) [inline tests/test_search.py]: no need to rename `search`, no built-in name `search`

**Comment by goswami-rahul** [comment]
@danghai this issue was faced before also. I fixed it for some other modules in #146 . So, it would be better if we follow the same convention and rename `array` -> `arrays`. I didn't know about `array` module at time, that's why we are facing this name conflict now. see #146 .
and no need for `search` (I don't think there's a module named search??)

**Comment by danghai** [comment]
I do not see any problem for `search` too, but #247 reports `search`

**Comment by danghai** [comment]
Okay , I will update and merge it

**Pull request merged**